src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -6676,6 +6834,20 @@ static void MT1(I1 a)
     }
 }
 ";
+
+            string matchingOp = MatchingBinaryOperator(op);
+
+            if (matchingOp is object)
+            {
+                source1 +=
+@"
+public partial interface I1
+{
+    abstract static I1 operator" + matchingOp + @" (I1 x, int y);
```

Comment:
Consider adding a bad `matchingOp` access as well.

Reply:
> Consider adding a bad matchingOp access as well.

That would duplicate the testing, we test all operators.